worker/internal/benthos/transformers/transform_character_substitution.go

```diff
@@ -48,52 +49,57 @@ Substituted: Ifmmp Xpsme 234@%^
 Note that this does not work for hex values: 0x00 -> 0x1F
 */
 
-func TransformCharacterSubstitution(value string) (*string, error) {
+func TransformCharacterScramble(value string) (*string, error) {
 
-	transformedString := strings.Map(substituteChar, value)
+	transformedString := strings.Map(ScrambleChar, value)
 
 	return &transformedString, nil
 }
 
-func substituteChar(r rune) rune {
-	letterMap := map[rune]rune{
-		'a': 'b', 'b': 'c', 'c': 'd', 'd': 'e', 'e': 'f', 'f': 'g', 'g': 'h', 'h': 'i', 'i': 'j',
-		'j': 'k', 'k': 'l', 'l': 'm', 'm': 'n', 'n': 'o', 'o': 'p', 'p': 'q', 'q': 'r', 'r': 's',
-		's': 't', 't': 'u', 'u': 'v', 'v': 'w', 'w': 'x', 'x': 'y', 'y': 'z', 'z': 'a',
-	}
+func ScrambleChar(r rune) rune {
+	letterList := "abcdefghijklmnopqrstuvwxyz"
```

Comment:
let's get these lists moved outside of the function in a module-level const block so they are only defined a single time.

Reply:
updated